lib/util.dart

```diff
@@ -119,4 +121,62 @@ class Util {
     fn(c);
     c.restore();
   }
+
+  /// Convert an array of pixel values into an [Image] object.
+  ///
+  /// The [pixels] parameter is the pixel data in the encoding described by
+  /// [PixelFormat.rgba8888], the encoding can't be changed to allow for web support.
+  ///
+  /// If you want the image to be decoded as it would be on the web you can set
+  /// [runAsWeb] to `true`. Keep in mind that it is slightly slower than the native
+  /// [ui.decodeImageFromPixels]. By default it is set to [kIsWeb].
+  Future<Image> decodeImageFromPixels(
```

Comment:
I think this method could be in the `Images` class instead, what do you think?

Reply:
Felt more a utility kind of method as we are basically wrapping existing functionality to allow for more possibilities, while the Images class is more meant to facilitate image loading stuff for Flame.

Reply:
I still think that this would fits better the images.dart, at the moment, I agree that our images.dart is only useful for loading images, but I think this could be easily expanded, and this method seems like a good one to start that expansion.

Also, we are doing some work for removing this file and moving the methods to places where makes more sense, so adding this method here seems like a move contrary to that work.

Reply:
You convinced me :-) I have updated the code.